packages/platform/src/internal/http/client.ts

```diff
@@ -118,7 +140,13 @@ export const fetch = (options?: RequestInit): Client.Client.Default =>
                       reason: "Transport",
                       error: _
                     })
-                })
+                }).pipe(
+                  disabledWhen(request as any) ?
+                    identity :
+                    Effect.withSpan(`http ${request.method}`, {
```

Comment:
One thing to think about here: the name of the span is the same for both inbound and outbound HTTP requests. Should we change them to `http.server` and `http.client`, respectively, so they can be distinguished?

Reply:
cc @tim-smart

Reply:
Went with `http.client`